launch_testing_ros/launch_testing_ros/test_runner.py

```diff
@@ -19,19 +19,4 @@
 
 
 class LaunchTestRunner(launch_testing.test_runner.LaunchTestRunner):
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        import rclpy  # Import on first use to avoid races at module level
-        self._rclpy_context = rclpy.context.Context()
-        rclpy.init(args=self._launch_file_arguments, context=self._rclpy_context)
-
-    def generate_preamble(self):
-        import launch_ros  # Import on first use to avoid races at module level
-        return [launch.actions.IncludeLaunchDescription(
-            launch_description_source=launch.LaunchDescriptionSource(
-                launch_description=launch_ros.get_default_launch_description(
-                    rclpy_context=self._rclpy_context,
-                )
-            )
-        )]
```

Comment:
This removed led to a new flake8 warning: https://ci.ros2.org/view/nightly/job/nightly_linux_debug/lastCompletedBuild/testReport/launch_testing_ros.test/test_flake8/test_flake8/

> `.\launch_testing_ros\test_runner.py:17:1: F401 'launch' imported but unused`

Reply:
Argh, I'll PR a fix ASAP.